docs/guides/building/cross-platform.md

```diff
@@ -105,6 +105,23 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install -y libgtk-3-dev webkit2gtk-4.0 libappindicator3-dev librsvg2-dev patchelf
+          
+      - name: Rust cache
+        uses: swatinem/rust-cache@v2
+        with:
+          workspaces: "./src-tauri -> target"
+          shared-key: ${{ runner.os }}-release
+
+      - name: Yarn cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/yarn
+            **/node_modules
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
```

Comment:
```suggestion
      - uses: actions/setup-node@v3
        with:
          node-version: 'lts/*'
          cache: 'yarn'
```

We may loose caching of the node_modules folder this way, but it looks way less intimidating for new-comers, and it's easier to change in case one uses npm or pnpm. what do you think?

Reply:
This is a pretty battle tested and quite default way to cache yarn, following best practices.

Personally I wouldn't change it to `setup-node` because its not caching the right files and it's much less descriptive (harder to recognise as a caching step).

Though if you do think it's better feel free to commit it.